src/NativeScript/Metadata/Metadata.mm

```diff
@@ -96,6 +96,35 @@ static int compareIdentifiers(const char* nullTerminated, const char* notNullTer
     return (MethodMeta*)result;
 }
 
+const std::vector<const MemberMeta*> BaseClassMeta::collectInheritanceChainMembers(const char* identifier, size_t length, MemberType type, bool includeProtocols, bool onlyIfAvailable, const ArrayOfPtrTo<MemberMeta>* members, int resultIndex) const {
```

Comment:
There's too much code duplication between this method and `members`. Try to eliminate it by implementing all cases of `members` in terms of the new `collectInheritanceChainMembers`.